ethereumj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -375,7 +376,7 @@ else if (oldValue != null && newValue.isZero()) {
             if (program.getNumber().intValue() == dumpBlock)
                 this.dumpLine(op, gasBefore, gasCost + callGas, memWords, program);
 
-            vmHookStepWrapper(program, op);
+            callVmHookAction(program, (hook, prg) -> vmHook.step(prg, op));
```

Comment:
`(hook, prg) -> vmHook.step(prg, op)` should be changed to `(hook, prg) -> hook.step(prg, op)` or you will call it twice in wrapper